src/comments/comments.service.ts

```diff
@@ -77,6 +93,7 @@ export class CommentsService {
                 id
             }
         })
-        return deletedComment;   
+        const action = await this.actionsService.addAction('Delete',userId,'Comment',comment.id,deletedComment)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them